src/controllers/customer.ts

```diff
@@ -5,6 +5,41 @@ import { LogToHelper } from '../middleware/auth/logto.js'
 
 export class CustomerController {
```

Comment:
There are there terms used, account, customer, client. Better to change everything to account

Reply:
Done! ✅